utils.py

```diff
@@ -176,8 +180,17 @@ def checkmate(board):
         color = board.black
     return (board.moves == [] and inCheck(board, color, enemy, board.isWhite, board.all))
 
-def remis(board):
-    return (board.halfmove == 50 or board.moves == [])
+def isRemis(board):
+    # TODO: Other Remis..
+    return (board.halfmove == 100 or board.moves == [])

```

Comment:
mach lieber >= 100 und warum checken wir board.moves == [] 2mal? wird ja auch in isMate gemacht

Reply:
`isMate` ist `true` wenn der Gegner im Schach ist und gleichzeitig keinen neuen Move mehr machen kann.
Remis gilt u.a. wenn der Gegner nicht im Schach ist aber trotzdem nichts machen kann (Stalemate).
Ich denke ich muss da noch ein `and not inCheck` hinzufügen